### ISSUE TITLE
DKP table validation seems to produce incorrect results

### ISSUE THREAD

**Issue by Cephel** [Roeshambo/MonolithDKP#302]
Greetings.

We are currently evaluating our tables because we found minor discrepancies (+-1 DKP in some players, between officers).

When we did a table validation, we noticed some rather large problems.

The most obvious example is this:
![image](https://user-images.githubusercontent.com/6716857/73122148-e0055b80-3f81-11ea-943e-1a2e80d7d597.png)
The player has only decay awards (he was added to the total list at some point but never gained or spent any DKP), yet his total DKP award came out to this after the validation:
![image](https://user-images.githubusercontent.com/6716857/73122157-fa3f3980-3f81-11ea-924e-84dc6497e06b.png)

We have no idea how and why this happens.

We repeated the validation step, with this result: 
![image](https://user-images.githubusercontent.com/6716857/73122162-17740800-3f82-11ea-82c0-87c396f54079.png)

**Comment by Roeshambo** [comment]
That's unusual. I'll have to look through the script and see why it might be doing that. It recreates each profile entry by entry chronologically and adds/subtracts as needed. It should be finding -10% of 0 is 0 obviously. Are there any other players in your table with a similar name or perhaps with a name that has "Arcum" in it?

**Comment by Cephel** [comment]
A search for "Arcum" only results in him, nobody else.

**Comment by Roeshambo** [comment]
There might be some funky math occurring in the WoW API when you attempt to get the percentage of 0 that I'll have to look into. If there are absolutely no entries containing his name aside from decays, it should just be subtracting 0 each time. The API has this weird way of deriving numbers (IE: if i put "1" into an edit box, the API will print out that number as something like 1.00302112562 or some crazy string of decimals)

**Comment by Cephel** [comment]
Yes there is absolutely no other mention of his name and his only awards are decays.

**Comment by Cephel** [comment]
We have more updated information: https://docs.google.com/document/d/1qLjaJHh48WXpoYxaI6cp555FiTyjxyfAMPJTEtqQ0GY/edit
